packages/kbn-i18n/README.md

```diff
@@ -72,19 +72,15 @@ export default function (kibana) {
 }
 ```
 
-The engine uses a locale resolution process similar to that of the built-in
-Intl APIs to determine which locale data to use based on the `accept-language`
-http header.
+The engine uses a `config/kibana.yml` file for locale resolution process.
```

Comment:
__note:__ this entire paragraph sounds a bit confusing. Maybe something like this?

> If locale is defined via `i18n.locale` option in `config/kibana.yml` then it will be used as a base locale, otherwise i18n engine will fall back to `en`. The `en` locale will also be used if translation can't be found for the base non-English locale.

> __Note:__ locale defined in `i18n.locale` and the one used for translation files should match exactly, e.g. `i18n.locale: zn` and `.../translations/zh_CN.json` won't match and default English translations will be used, but `i18n.locale: zh_CN` and `.../translations/zh_CN.json` or `i18n.locale: zn` and `.../translations/zn.json` will work as expected.